test/extended/testdata/builds/build-postcommit/docker.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: v1
+kind: List
+metadata: {}
+items:
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    name: mydockertest
+    labels:
+      name: test
+  spec:
+    triggers: []
+    runPolicy: Serial
+    source:
+      dockerfile:
+        'FROM openshift/hello-openshift:latest'
+    strategy:
+      type: Docker
+      dockerStrategy:
+    output:
```

Comment:
you can speed these builds/tests up by nil'ing the output field and only setting it on the test where you actually need the output image (the last test where you are checking for the presence of the file)

Reply:
Updated in current push, thanks!

Reply:
Seems to have shaved off about one minute